plugins/experimental/ats_pagespeed/Makefile.am

```diff
@@ -0,0 +1,72 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+include $(top_srcdir)/build/plugins.mk
+
+if ATS_PAGESPEED_LINUX
+  os_name = linux
+else
+  os_name = unknown_os
+endif
+
+if ATS_PAGESPEED_X86_64
+  arch_name = x64
+else
+  arch_name = ia32
+endif
+
```

Comment:
You don't need to do this. In ``configure.ac`` changes, just ``AC_SUBST`` the values. Then later on you can use ``@ATS_PAGESPEED_OS@`` and ``@ATS_PAGESPEED_ARCH@``.